bindings/java/Makefile

```diff
@@ -22,6 +22,9 @@ $(OUT_DIR)/lib/libevmc-java.so:
 	mkdir -p $(BUILD_DIR)
 	(cd $(BUILD_DIR) && cmake $(SOURCE_DIR) -DCMAKE_INSTALL_PREFIX=$(OUT_DIR) -DEVMC_JAVA=ON -DJAVA_HOME=$(JAVA_HOME) -DEVMC_EXAMPLES=ON)
 	cmake --build $(OUT_DIR)/_cmake_build --target install
+	mkdir -p $(CURDIR)/java/src/test/resources
+	cp $(OUT_DIR)/lib/libevmc-java.so $(CURDIR)/java/src/test/resources/libevmc.so
+	cp $(OUT_DIR)/lib/libexample-vm.so $(CURDIR)/java/src/test/resources/libexample-vm.so
```

Comment:
Copy files over to test folder so it gets picked up by tests.

Reply:
To clarify, this change was made in support of this:


https://github.com/ethereum/evmc/pull/532/files/0c9eb8e30b24e18e2910aef98c8a84c33d1360a8#r464218230

right?

Reply:
Indeed.